project/src/main/webapp/index.html

```diff
@@ -17,22 +24,50 @@
     <title>Voter Central Homepage</title>
   </head>
   <body>
-    <header></header>
-    <nav-bar></nav-bar>
-    <main role="main">
-      <form id="location-input" action="/directory.html">
-        <label for="address">Enter your street address:</label>
-        <br>
-        <input type="text" id="address" name="address" placeholder="...your street address..."
-                required/>
-        <input type="hidden" name="listAllElections" value="false"/>
-        <input type="submit" id="submit-address"/>
-      </form>
-      <form id="list-all-input" action="/directory.html">
-        <input type="hidden" name="address" value="default"/>
-        <input type="hidden" name="listAllElections" value="true"/>
-        <input type="submit" id="submit-list-all" value="List All Elections"/>
-      </form>
-    </main>
+    <div class="container">
+      <div class="row justify-content-end">
+      <div class="col-9">
+          <header></header>
+      </div>
+      </div>
+      <div class="row">
+        <div class="col">
+          <nav-bar></nav-bar>
+        </div>
+        <div class="col-9">
+          <main role="main">
+            <form id="location-input" action="/directory.html">
+              <label for="address">Enter your street address:</label>
+              <br>
+              <input type="text" id="address" name="address" placeholder="...your street address..."
+                      required/>
+              <input type="hidden" name="listAllElections" value="false"/>
+              <input type="submit" id="submit-address"/>
+            </form>
+            <form id="list-all-input" action="/directory.html">
+              <input type="hidden" name="address" value="default"/>
+              <input type="hidden" name="listAllElections" value="true"/>
+              <input type="submit" id="submit-list-all" value="List All Elections"/>
+            </form>
+          </main>
+        </div>
+      </div>
+    </div>
+    <script src="https://code.jquery.com/jquery-3.5.1.slim.min.js" integrity=
```

Comment:
You're reusing the various script srces a lot. Consider making them shared variables?